packages/pros/src/adi/potentiometer.rs

```diff
@@ -0,0 +1,35 @@
+use crate::adi::{
+    AdiError,
+    AdiSlot
+};
+
+use pros_sys::PROS_ERR;
+
+use crate::error::bail_on;
+
+pub struct AdiPotentiometer {
+    port: u8,
+    reference: i32
```

Comment:
Naming: (`reference` -> `raw`). This should also probably be `pros_sys::adi_potentiometer_t` for clarity.